beancount_dkb/ec.py

```diff
@@ -147,10 +149,18 @@ def extract(self, file_, existing_entries=None):
                             )
                         )
                 else:
-                    description = '{} {}'.format(
-                        line['Buchungstext'],
-                        line['Verwendungszweck'] or line['Kontonummer'],
-                    )
+                    verwendungszweck = line['Verwendungszweck'] or \
+                        line['Kontonummer']
+                    buchungstext = line['Buchungstext']
+
+                    if self.meta_code:
+                        meta[self.meta_code] = buchungstext
+                        description = verwendungszweck
```

Comment:
Is there a specific reason why `description` is different in both cases? Could we keep it the same for when `meta_code` is present/absent?

Reply:
The reason it is different is that I don't add "Buchungstext" if the meta tag is used. This would be quite redundant and add unnecessary noise in the transaction logs that could impact readability. I know that it adds complexity to the code, but I think the result is more desirable.